smarty/tests/test_sampler.py

```diff
@@ -57,14 +56,14 @@ def test_atomtyper_elemental(self):
         """
         Test elemental atomtype sampler for hydrogen
         """
-        atomtype_sampler = smarty.AtomTypeSamplerElemental(self.mols_alkethoh,
+        atomtype_sampler = smarty.AtomTypeSampler(self.mols_alkethoh,
                 self.basetypes, self.basetypes, self.combine_decs,
                 self.replacements, self.mols_alkethoh_ref, 0.1, False,
-                'combinatorial-decorators', '1')
+                'combinatorial-decorators', 1)
```

Comment:
@davidlmobley 
The function is element=0 as a default, I thought if you called a function with the keyword arguments in order you didn't have put the name in front? 
Is there a style preference for this?
The decorator type, temperature, verbose setting, and replacements are also keyword arguments.